### DISCUSSION TITLE
detect velocity estimation using yolov5

### DISCUSSION THREAD

**Discussion by andiandhika** [ultralytics/yolov5#8639] [Q&A]
How do we determine the object in front of the camera is moving or our camera is moving?, because if we enter the position/time difference then miscalculation may occur (for example if the speed of the vehicle mounted on the camera (object) is the same as the speed of the vehicle in front (object), in the video captured automatically there is never a difference
distance, then the possible speed of the vehicle in front can be calculated 0, even though it is not)

**Comment by glenn-jocher** [comment]
@andiandhika I think your question is outside the scope of YOLOv5 support. Naturally the velocity of the observer would be taken into account if you expect an absolute velocity measurement of an object. This is simple physics.
